index.html

```diff
@@ -301,6 +302,7 @@
 <body>
```

Comment:
I suggest to add the following style 
```css
	#docs-footer {
		text-align: center;
		background-color: var(--color-1);
		color: white;
		padding: 1rem;
	}

	#docs-footer a {
		color: white;
		text-decoration: underline;
	}
```

![image](https://github.com/arnaudsm/raito/assets/3920793/3ffe16fe-0d5d-4dfa-ad3c-f46cc469e6a1)

Reply:
i want to congratulate you for the simplicity, each style is simply the name of the file with its path... wonderful @arnaudsm

Reply:
done with commit https://github.com/arnaudsm/raito/pull/48/commits/6a5204846eb0380ecd3ea04bed595ea1256b0798